prescient/simulator/tests/test_simulator.py

```diff
@@ -117,7 +111,7 @@ def _assert_column_equality(self, filename, column_name):
         df_a = self.test_results[filename]
         df_b = self.baseline_results[filename]
         dtype = df_a.dtypes[column_name]
-        if dtype == 'float' or dtype == 'int':
+        if dtype == 'float' or dtype == 'int' or dtype == 'int64' or dtype == 'float64':
```

Comment:
```suggestion
        if dtype.kind in "iuf":
```
May be better to just ask about its kind? https://numpy.org/doc/stable/reference/generated/numpy.dtype.kind.html#numpy.dtype.kind